packages/zowe-explorer/__tests__/__integration__/bdd/features/dialogs/ShowConfigErrorDialog.feature

```diff
@@ -0,0 +1,7 @@
+Feature: Show Config Error Dialog
+
+Scenario: Initializing Zowe Explorer with a broken profile
+    When a user opens Zowe Explorer
+    Then the Show Config dialog should appear
```

Comment:
I'm afraid that this step didn't work (as-is).
I had to introduce an error in the config file in order for the test to pass.

Curious if we should add a few steps:
```suggestion
    When a user selects "Edit Team Configuration File" # existing step
    Then it will open the config in the editor #existing step
    When the user introduces and error and saves the configuration file # new step
    Then the Show Config dialog should appear
```

Reply:
Thanks for the quick discussion offline 🙏